src/ImageRunModal.jsx

```diff
@@ -157,6 +165,7 @@ export class ImageRunModal extends React.Component {
             healthcheck_timeout: 30,
             healthcheck_start_period: 0,
             healthcheck_retries: 3,
+            healthcheck_action: 0,
```

Comment:
I still object to this default -- "No action" is completely pointless for a watchdog. "Force stop" may be a better default. What is the default on main where users can't configure the action type?

Reply:
This is the default of podman itself, should we really override that behaviour. It doesn't sound correct and it would be a breaking change.

https://docs.podman.io/en/latest/markdown/podman-run.1.html#health-on-failure-action

Reply:
Hm, this really leaves us between a rock and a hard place. It makes sense to keep the UI compatible with the CLI. But it does tell me that our current feature in `main` is broken: you can configure a health check command, and then it is a complete no-op? Likewise with the CLI, if you do a `--health-cmd`, it goes entirely ignored? That is at least worth a podman bug report.

I have no strong opinion between defaulting to "kill" (and configuring a container to auto-spawn, like on k8s), or "restart" (which may make more sense without wrapping in systemd).

Reply:
So in the interest of not blocking this PR, we can do this as a follow-up. But I am really not happy about this..